notebooks/nb_utils/lesson_preprocessor.py

```diff
@@ -84,26 +98,32 @@ def pip_install_hack(self, nb, pkgs):
                 'import sys\n',
                 "sys.path.append('/kaggle/working')",
         ]
-        syspath_cell = self.make_code_cell(source=syspath_lines)
+        syspath_cell = self.make_cell(cell_type='code', source=syspath_lines)
         extra_cells.append(syspath_cell)
         nb.cells = extra_cells + nb.cells
 
     @classmethod
     def pip_install_cell(cls, pkg_spec):
         cmd = '!pip install -U -t /kaggle/working/ {}'.format(pkg_spec)
-        return cls.make_code_cell(source=[cmd])
+        return cls.make_cell(cell_type='code', source=[cmd])
 
     @staticmethod
-    def make_code_cell(**kwargs):
+    def make_cell(cell_type, **kwargs):
+        """Returns a NotebookNode object populated with kwargs. cell_type should be either 'markdown' or 'code'"""
```

Comment:
You could raise an error if `cell_type` is not `markdown` or `code`.  Enums would also be good here.